docs/optimization-framework/experience-blocks.md

```diff
@@ -2,11 +2,11 @@
 
 Altis [Experience Blocks](https://www.altis-dxp.com/experience-blocks/) (XBs) are built-in blocks for the editor that provide advanced capabilities. We are continually extending the range and uses of these blocks.
 
-The blocks themselves are backed by analytics data and [audiences](./audiences.md) making it possible to record interactions and impressions with specific pieces of content on your pages. This document outlines the underlying mechanism so that you can implement these blocks outside of the scope of the block editor.
+The blocks themselves are backed by analytics data and [audiences](../native/audiences.md) making it possible to record interactions and impressions with specific pieces of content on your pages. This document outlines the underlying mechanism so that you can implement these blocks outside of the scope of the block editor.
 
-## Personalization
+## Personalization API
 
-Altis empowers editorial teams to create personalized content with the Personalized Content Experience Block. Because it's crucial that web pages can be fully cached for performance, our Personalized Content blocks use [web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) that are simple to use and load immediately as the document is loaded.
+Altis empowers editorial teams to create personalized content with the Personalized Content Experience Block. And because it's crucial that web pages can be fully cached for performance, our Personalized Content blocks use [web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) that are simple to use and load immediately as the document is loaded.
```

Comment:
Remember, don't start a sentence with "And", either switch to a comma or leave as-is. I see how it could perhaps read as a direct follow on from the first sentence but the full stop was intentional. How about this small tweak:

```suggestion
Altis empowers editorial teams to create personalized content with the Personalized Content Experience Block. Because it is crucial that web pages can be fully cached for performance, our Personalized Content blocks use [web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) that are simple to use and load immediately as the document is loaded.
```